src/sdk/models/users.ts

```diff
@@ -10,8 +10,9 @@ export interface User extends IEntityIdentifier<string> {
 	name?: string;
 	email?: string;
 	disabled?: boolean;
+	readonly?: boolean;
```

Comment:
Not sure if I'm missing something, but `SecurityUser` doesn't have a `readonly` property

Reply:
Ah nice catch. This should be modifying [Account](https://github.com/portalarchitects/skysync-cli/blob/master/src/sdk/models/connections.ts#L98) not `User`.

Reply:
That inherits from `SecurityIdentifier`, which already has it - https://github.com/portalarchitects/skysync-cli/blob/master/src/sdk/models/connections.ts#L90